react-native-libraries.json

```diff
@@ -7592,7 +7593,13 @@
       "https://raw.githubusercontent.com/esthor/react-native-swipeable-list/master/images/react-native-swipeable-list-demo2.gif"
     ],
     "ios": true,
-    "android": true
+    "android": true,
+    "web": true,
+    "expoGo": true,
+    "unmaintained": false,
+    "dev": false,
+    "template": false,
```

Comment:
All `false` flags can be skipped:
* https://github.com/react-native-community/directory?tab=readme-ov-file#new-library-entry-template

Reply:
Oh, my bad. Didn't read the docs 🙄